conrado/script.js

```diff
@@ -0,0 +1,49 @@
+function randomColorRGB() {
+    var num1 = Math.floor(Math.random() * 256);
+    var num2 = Math.floor(Math.random() * 256);
+    var num3 = Math.floor(Math.random() * 256);
+    var bgColor = "rgb(" + num1 + "," + num2 + "," + num3 + ")";
+    let rgb = document.getElementById("numbers");
+    rgb.innerHTML = bgColor;
+    return bgColor;
+}
+
+randomColorRGB();
+function randomColor(){
+    let colors;
+    for (i = 0; i <= 5; i++){
+        colors = document.getElementsByClassName("add")[i];
```

Comment:
Você pode coloacar essa linha antes do `for`, de forma que não vai precisar buscar os elementos no DOM a cada iteração

```suggestion
        let colors = document.getElementsByClassName("add");
        
        for (color of colors) {
          colors.style.background = randomColorRGB();
        }
        
```